### PR TITLE
Exclude templates from in-active channels

### PR DESCRIPTION
**Pull request by smn**
This addresses part of https://github.com/rapidpro/rapidpro/issues/1114 by making sure the queryset returned is correct. However the prefetched results are ignored when serialised to JSON as the translation for the in-active channel is still being returned in the results.

### CODE REVIEW

**Comment by smn** [comment]
Adding the filter to the `TemplateReadSerializer` seems to resolve the issue but that means we're doing this filtering in two locations rather than one, which is less than ideal.

**Comment by smn** [comment]
the last build failure looks unrelated to these changes, looks like a race condition somewhere.

**Comment by ericnewcomer** [comment]
I restarted the failed build and it passed, so it does indeed look like an intermittent failure.

File: temba/api/v2/serializers.py [hunk 1]
```diff
@@ -1298,7 +1298,9 @@ class TemplateReadSerializer(ReadSerializer):
     def get_translations(self, obj):
         translations = []
         for translation in (
-            TemplateTranslation.objects.filter(template=obj).order_by("language").select_related("channel")
+            TemplateTranslation.objects.filter(template=obj, channel__is_active=True)
```
Comment (rowanseymour) [inline temba/api/v2/serializers.py]: should we be de-activating template translations when a channel is de-activated?

Comment (nicpottier) [reply temba/api/v2/serializers.py]: We do currently: https://github.com/nyaruka/rapidpro/blob/master/temba/channels/types/whatsapp/type.py#L139

Problem is we aren't filtering by `is_active` in the API views which I think is the fix this PR is making.

Comment (rowanseymour) [reply temba/api/v2/serializers.py]: so shouldn't this be filtering by TemplateTranslation.is_active ?

Comment (nicpottier) [reply temba/api/v2/serializers.py]: Oh I see, yes, you are totally right. This should just be filtering by `is_active` on TemplateTranslation.

**Comment by smn** [comment]
thanks for the review @ericnewcomer @rowanseymour and @nicpottier, I've addressed the issues. Please have another look, thanks!

**Pull request merged**